core/db/default/spree/countries.rb

```diff
@@ -1,17 +1,13 @@
 require 'carmen'
 
-countries = []
-Carmen::Country.all.each do |country|
-  countries << {
-    name: country.name,
-    iso3: country.alpha_3_code,
-    iso: country.alpha_2_code,
-    iso_name: country.name.upcase,
-    numcode: country.numeric_code,
-    states_required: country.subregions?
-  }
-end
-
 ActiveRecord::Base.transaction do
-  Spree::Country.create!(countries)
+  Carmen::Country.all.each do |country|
+    Spree::Country.where(iso: country.alpha_2_code).first_or_create(
```

Comment:
Could this be `first_or_create!` so that we receive errors if there are any problems?